### PR TITLE
PPF-485 mass unblock keycloak status

### PR DESCRIPTION
**Pull request by grubolsch**
### Added
When you unblock an entire integration, also unblock all Keycloak clients.

---
Ticket: https://jira.uitdatabank.be/browse/PPF-485

### CODE REVIEW

File: app/Keycloak/Listeners/UnblockClients.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                ]);
+            } catch (KeyCloakApiFailed $e) {
+                $this->failed($integrationUnblocked, $e);
+            }
+        }
+    }
+
+    public function failed(IntegrationUnblocked $integrationUnblocked, Throwable $throwable): void
+    {
+        $this->logger->error('Failed to block Keycloak client(s)', [
```
Comment (JonasVHG) [inline app/Keycloak/Listeners/UnblockClients.php:51]:
```suggestion
        $this->logger->error('Failed to unblock Keycloak client(s)', [
```
Probably a small copy-paste error.

**Review by JonasVHG — APPROVED** [review]
There is 1 small typo that should be fixed, otherwise it looks good.

**Comment by LucWollants** [comment]
@grubolsch I wonder how this listener is bound to the event?

**Comment by grubolsch** [comment]
@LucWollants Good point, forgot to git add that change! Fixed this.

**Pull request merged**